app/models/observers/notice_observer.rb

```diff
@@ -59,13 +59,31 @@ def plan_year_update(new_model_event)
               end
             end
         end
+        
+        if new_model_event.event_key == :initial_employer_denial
+          return true if plan_year.benefit_groups.any?{|bg| bg.is_congress?}
+          if (plan_year.application_eligibility_warnings.include?(:primary_office_location) || plan_year.application_eligibility_warnings.include?(:fte_count))
+            begin
+              trigger_notice(recipient: plan_year.employer_profile, event_object: plan_year.employer_profile, notice_event: "initial_employer_denial")
```

Comment:
You might want to send plan year event object instead of employer profile because we need to populate plan years application eligibility warnings on the notice.